app/components/blacklight/search_context/server_item_pagination_component.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Blacklight
+  module SearchContext
+    class ServerItemPaginationComponent < Blacklight::SearchContextComponent
```

Comment:
I like this approach of inheriting from the Blacklight 7 component, and providing an interface that is compatible with the Blacklight 8 component!